admin/client/views/home.js

```diff
@@ -19,16 +21,17 @@ var ListTile = React.createClass({
     count: React.PropTypes.string,
     href: React.PropTypes.string,
     label: React.PropTypes.string,
+    noEdit: React.PropTypes.bool,
```

Comment:
Question:
What is the purpose of `noEdit` props ?

Reply:
![Screen Shot 2023-01-20 at 10 21 06 PM](https://user-images.githubusercontent.com/25971696/213721761-4e46c100-1c7f-4b14-af21-88a939ea6d44.png)
`noEdit` prop is to disable the `add` button in latest tile